build-tools/automation/build.groovy

```diff
@@ -145,96 +228,100 @@ timestamps {
         }
 
         stageWithTimeout('Plot build & test metrics', 30, 'SECONDS', XADir, false) {    // Typically takes less than a second
-            plot(
```

Comment:
Does Groovy have an "early return" concept, so that we could "check & bail" within this block without needing to reindent everything?

Reply:
This answer: https://stackoverflow.com/a/1343703/83444

Suggests that we might be able to use `return`:

```groovy
if (isPr) {
    echo "Skipping plot metrics for PR build"	
    return
}
```

Reply:
Yup, we can use return. Fixed with commit https://github.com/xamarin/xamarin-android/pull/2767/commits/49c767430c5334374412d168a13e1b6b9d96332b